ibazel/BUILD

```diff
@@ -35,7 +41,6 @@ go_library(
         "//ibazel/command:go_default_library",
         "//third_party/bazel/master/src/main/protobuf:go_default_library",
         "@com_github_fsnotify_fsnotify//:go_default_library",
-        "@com_github_gorilla_websocket//:go_default_library",
         "@com_github_gregmagolan_lrserver//:go_default_library",
```

Comment:
This can point to the upstream lrserver again. https://github.com/jaschaephraim/lrserver. My changes to add Close() and Shutdown() were merged in.

Is the gorilla websocket dep not needed here?

Reply:
TBH I ran Gazelle on it and this is what came out. I guess it isn't :man_shrugging:

Reply:
Forgot to respond to the first point. Yep, I'll change those in the next commit :smile: